src/miner/miner.rs

```diff
@@ -42,7 +47,7 @@ impl Miner {
     config: &PowConfig,
   ) -> Result<(), Error> {
     let block: Block = service.get_block(&block_id)?;
-    let header: BlockHeader = BlockHeader::borrowed(&block).expect("Block Header");
+    let header: BlockHeader = BlockHeader::borrowed(&block).expect("Block head Header");
```

Comment:
I'm a little confused by "block head header" is it a typo, or what is a "block head"?

Reply:
Yup, that's not very descriptive. What about Chain head header?